js/jest_get_defaults.js

```diff
@@ -0,0 +1,29 @@
+/* Systematic's jest base config file */
+// see https://facebook.github.io/jest/docs/en/configuration.html
+
+'use strict'
+
+module.exports = function (basePath) {
+  return {
+    moduleFileExtensions: [
+      'js',
+      'json',
+      'vue',
+    ],
+    transform: {
+      '.*\\.(vue)$': `${basePath}/node_modules/vue-jest`,
+      '^.+\\.js$': `${basePath}/node_modules/babel-jest`,
+    },
+    transformIgnorePatterns: [
+      'node_modules/(?!(cassets|vanitils)/)',
```

Comment:
this is our internal libraries

Reply:
what if we just dropped karma, and also move the further Systematic developments to internal ? I'm not sure anyone really uses this project anyway.